lib/components/map/set-from-to.js

```diff
@@ -18,19 +18,7 @@ export default class SetFromToButtons extends Component {
 
   render () {
     return (
-      <div style={{ display: 'inline-block' }}>
-        <span><b>Plan a trip: </b></span>
-        <Button bsSize='xsmall' className='set-from-button'
-          onClick={this._setFromClicked}
-        >
-          <LocationIcon type='from' /> From here
-        </Button>
-        <Button bsSize='xsmall' className='set-to-button'
-          onClick={this._setToClicked}
-        >
-          <LocationIcon type='to' /> To here
-        </Button>
-      </div>
+      <FromToLocationPicker onFromClick={this._setFromClicked} onToClick={this._setToClicked} />
```

Comment:
Is there a way to completely replace the `SetFromToButtons` component with the `FromToLocationPicker`? The `FromToLocationPicker` is able to call a `setLocation` method with the same arguments that are in here.

Reply:
I will decline for now. We will need our own Popup class for use inside overlays and inside `BaseMap` before we can inline `SetFromToButtons`.  Right now, `SetFromToButtons` takes in a map so it can close the popup when you click on the links.